doozerlib/cli/images_upstreampulls.py

```diff
@@ -0,0 +1,62 @@
+# stdlib
+import os
+import yaml
+
+# external
+from github import Github
+
+# doozerlib
+from doozerlib import constants
+from doozerlib.cli import cli, pass_runtime
+
+github_token = os.getenv(constants.GIT_TOKEN)
+github_client = Github(github_token)
+repo = github_client.get_repo("openshift/ocp-build-data")
+
+
+@cli.command("images:upstreampulls", short_help="Check upstream PRs in open status")
```

Comment:
It doesn't seem to me that the verb name `images:upstreampulls` or the help message explains well about what it  intends to do. Would like to have a better verb name like `images:streams prs list` for consistency and improved help message.